packages/hash/external-services/kratos/templates/recovery_code/invalid/email.body.plaintext.gotmpl

```diff
@@ -0,0 +1,9 @@
+Hi,
+
+you (or someone else) entered this email address when trying to recover access to an account.
```

Comment:
I think it would make it less ambiguous for now what these emails are referring to if we refer to a "HASH account" rather than just "account". (this suggestion applies to most templates)
```suggestion
you (or someone else) entered this email address when trying to recover access to a HASH account.
```

cc @nonparibus for thoughts on the initial phrasing here

Reply:
Agreed "HASH account" preferable.

I would like to do a big pass at all of our transactional email copy soon.

Not necessarily as part of this PR, but could we consider how we might centrally collect all of our transactional email text in one place?

This could be creating an index pointing to the relevant lines of files like this, it could be a single directory that contains the actual emails themselves, or something like else (better) entirely. Either way we're not we'll set up for this at the moment.

Reply:
@nonparibus for now all the kratos transactional emails are in this folder https://github.com/hashintel/hash/tree/e98a86be821317929dfaa15edfd255af51222726/packages/hash/external-services/kratos/templates
for emails that we will send through HASH, we have to consider what we do to either co-locate them together or make them easily searchable. For now I think this kratos grouping is a nice start

Reply:
Just want to flag that this suggestion also applies to the remaining email templates (including email subjects). Let me know if you intend on addressing this @thehabbos007

Reply:
Sorry, missed that this was for all: f710e62f0ba6e547b248e509a1cb59f9c24fb39f

Reply:
Thanks -- wasn't aware we had the dir already. This is great.

I don't know what ultimate best-practice looks like here, but I would be happy with either:
- somewhere I can go that links out to all the different collections of transactional emails (like this), provided we actually keep it up to date;
- a single directory in which we store all transactional emails from across the app.

My sense is the latter probably makes marginally more sense, and will be easier to track.

I'll review/update Kratos email copy in a separate PR. Please make any adjustments between the two of you that you think make sense in the meantime.

Reply:
Changed in 8ff88d9d44cff7c81be30829c55b8ea2b39cc691
Index at https://github.com/hashintel/hash/blob/8ff88d9d44cff7c81be30829c55b8ea2b39cc691/packages/hash/README.md#sending-emails